.changeset/sharp-chicken-swim.md

```diff
@@ -0,0 +1,5 @@
+---
+'@prosekit/react': patch
+---
+
+add table icon in popover
```

Comment:
You don't need to add a changeset file because the change included in this PR won't be published to NPM (i.e. `packages/react/` is not touched).

Reply:
thanks, it has been removed.